portal/tests/test_security.py

```diff
@@ -100,3 +100,8 @@ def test_reminder_cards_wrong_teacher(self):
     def test_class_page_wrong_teacher(self):
         """Try and view a class page without being the teacher for that class."""
         self._test_incorrect_teacher_cannot_login('onboarding-class')
+
+    def test_anonymous_cannot_access_teaching_materials(self):
+        c = Client()
+        page = reverse_lazy('materials')
+        self.assertNotEqual(c.get(page).status_code, 200)
```

Comment:
This doesn't really test for authentication, instead, maybe assert for `401` , `403` or whatever else we expect?

Reply:
true